### PR TITLE
JBPM-4625 Generation of JPA enabled models

### PR DESCRIPTION
**Pull request by wmedvede**

### CODE REVIEW

File: drools-wb-webapp/pom.xml [hunk 1]
```diff
@@ -792,7 +797,9 @@
         <configuration>
           <deploy>${project.build.directory}/gwt-symbols-deploy</deploy>
           <localWorkers>1</localWorkers>
-          <extraJvmArgs>-Xmx2048m -XX:MaxPermSize=256m -Xms1024m -XX:PermSize=128m -Xss1M -XX:CompileThreshold=7000 -Derrai.jboss.home=${errai.jboss.home}</extraJvmArgs>
+          <extraJvmArgs>-Xmx2048m -XX:MaxPermSize=256m -Xms1024m -XX:PermSize=128m -Xss1M -XX:CompileThreshold=7000 -Derrai.jboss.home=/home/wmedvede/development/projects/droolsjbpm/drools-wb/drools-wb-webapp/target/wildfly-8.1.0.Final</extraJvmArgs>
```
Comment (manstis) [inline drools-wb-webapp/pom.xml]: Tssk tssk.. I don't want to create this path on my laptop ;)

Comment (manstis) [inline drools-wb-webapp/pom.xml]: I've replaced with ${errai.jboss.home} in the merge.

**Comment by manstis** [comment]
http://github.com/droolsjbpm/drools-wb/commit/98f8e709f

File: drools-wb-webapp/pom.xml (hunk 1 above)
Comment (Rikkola) [inline drools-wb-webapp/pom.xml]: I actually did the same change. Based on Porcelli's instructions.
But once I got everything working and tried with the variable. There actually is no need to change this line in order to get the super dev mode working. It is enough to set this variable in the Idea's run configurations.

**Pull request closed without merging**